### DISCUSSION TITLE
Pole figure program issue (Zirconium)

### DISCUSSION THREAD

**Discussion by Ashley1892** [mtex-toolbox/mtex#1531] [Ask Anything]
Hi all,
I'm trying to read .uxd XRD data into MTEX to get pole figures. When I go through the import wizard, I can get plotted data. However, if I run the produced program in MatLab (ultimately with the goal of constructing ODFs from these pole figures) I get the attached message. The material I am using is (mostly) zirconium, and the planes I'm looking at are the zirc planes. Anyone seen this before?
![image](https://user-images.githubusercontent.com/111444689/185169563-5d553191-a209-43d4-807e-3bf4acad2a74.png)

**Comment by vtvivian** [comment]
Hi! Your CS variable import options look wrong! XYZ vectors need to be perpendicular to each other (this is your XRD measurement coordinates). Hexagonal crystals such as Zr have 120 degrees between <a> and <b>, and there are usually 2 conventions for lining this up with the XYZ coordinates. You have to find out which convention your measurement output data follows.

(1) X||a*, Y||b, Z||c (this means <b> lines up with Y, and <a> is -30 degrees from X)
(2) X||a, Y||b*, Z||c [this means <a> lines up with X, and <b> is 30 degrees from Y)

X||a* means X is perpendicular to <b> and <c>
Y||b* means Y is perpendicular to <c> and <a>
Z||c* and Z||c are equivalent in Zr (but not necessarily other crystals!)
 
what you have is  X||a*, Y||b*, Z||c which means your XYZ is not perpendicular to each other, which is why you get this error.
